### DISCUSSION TITLE
How to use multiline Input

### DISCUSSION THREAD

**Discussion by freva** [mantinedev/mantine#4236] [Q&A] [answered]
Mantine input [documents](https://mantine.dev/core/input/?t=props) a `multiline` option, but I cant see any difference when enabling it on my long text, example: https://codesandbox.io/s/mantine-core-multiline-input-6fqrml

How is this option meant to be used?

**Comment by rtivital** [comment, marked as answer]
https://codesandbox.io/s/mantine-core-multiline-input-forked-tu7zq7

**Reply by freva** [reply]
Perfect, thanks!
